### PR TITLE
Fix layout for roadmap on mobile screen

### PR DESCRIPTION
**Pull request by piotrpasich**
image

Fixes the  https://github.com/x-team/auto/issues/787

### CODE REVIEW

File: src/styles/typo.js [hunk 1]
```diff
@@ -184,6 +194,8 @@ export default {
     @media screen and (max-width: ${breakpoints.sm}) {
       & {
         width: 2rem
+        margin-left: auto
```
Comment (nicksp) [inline src/styles/typo.js]: That's no-no. Divider should only be centered if centered prop is passed to the text component. In other cases I don't think we're able to force each instance to be centered

Comment (piotrpasich) [reply src/styles/typo.js]: ok

**Comment by nicksp** [comment]
@piotrpasich Do you need any help mate with progressing with this one? Just let me know if so 😏

**Comment by piotrpasich** [comment]
@nicksp fixed

File: src/components/ui/RoadmapHero.js [hunk 2]
```diff
@@ -52,14 +62,15 @@ class RoadmapHero extends PureComponent<Props, State> {
...
+    this.updateAlignment()
   }
 
   componentWillUnmount () {
     window.removeEventListener('resize', this.updateAlignment)
   }
 
   updateAlignment = () => {
-    const mq = window.matchMedia(`(max-width: ${breakpoints.xs})`)
+    const mq = window.matchMedia(`(max-width: ${breakpoints.lg})`)
```
Comment (nicksp) [inline src/components/ui/RoadmapHero.js]: this should still be `xs` as we only want to center on mobile, and keep it left aligned on other screens.

**Comment by nicksp** [comment]
Updated some files. Merging now

**Pull request merged**